### PR TITLE
US44882: Fixes

### PR DESCRIPTION
**Pull request by sgawroriskiPGH**
This will be a PR for fixes to the project, including a fix to correct delimiter issues.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -316,3 +345,4 @@ JUnitXmlPlugin.prototype.teardown = async function () {
 
 module.exports = new JUnitXmlPlugin();
 module.exports.JUnitXrayPlugin = JUnitXmlPlugin;
+module.exports.findXrayIdAndName = findXrayIdAndName;
```
Comment (arsal123) [inline index.js]: Do we need to export this?

File: package.json [hunk 2]
```diff
@@ -24,7 +28,7 @@
     "protractor": "5.4.2"
   },
   "scripts": {
-    "test": "echo 'no tests yet'"
+    "test": "./node_modules/mocha/bin/mocha tests/*.js --reporter spec"
```
Comment (arsal123) [inline package.json:32]: Thats a good addition. Thanks

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+name: Testing
+
```
Comment (arsal123) [inline .github/workflows/test.yml:2]: :+1:

File: index.js (hunk 1 above)
Comment (sgawroriskiPGH) [reply index.js]: I moved it to a `common.js` file and imported it in the plugin and unit tests.

**Review by arsal123 — APPROVED** [review]

**Pull request merged**